Easydict/NewApp/Utility/GlobalContext.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  GlobalContext.swift
+//  Easydict
+//
+//  Created by 戴藏龙 on 2024/1/25.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Foundation
+import Sparkle
+
+@objc class GlobalContext: NSObject {
+    /// Initialized all property in global context to assure static properties life circle.
+    static func initializeProperty() {
+        let _ = Self.updaterController
+    }
+
+    static let updaterController: SPUStandardUpdaterController = {
+        class SPUUpdaterHelper: NSObject, SPUUpdaterDelegate {
+            func feedURLString(for _: SPUUpdater) -> String? {
+                var feedURLString = "https://raw.githubusercontent.com/tisfeng/Easydict/main/appcast.xml"
+                #if DEBUG
+                    feedURLString = "http://localhost:8000/appcast.xml"
+                #endif
+                return feedURLString
+            }
+        }
+
+        class SPUUserDriverHelper: NSObject, SPUStandardUserDriverDelegate {
+            var supportsGentleScheduledUpdateReminders: Bool {
+                true
```

Comment:
The new updater delegate method doesn't work, still getting a warning, as well as `func feedURLString(for _: SPUUpdater) -> String?` above.

```
Warning: Background app automatically schedules for update checks but does not implement gentle reminders. As a result, users may not take notice to update alerts that show up in the background. Please visit https://sparkle-project.org/documentation/gentle-reminders for more information. This warning will only be logged once.
```